examples/nextjs-example/pages/headless.tsx

```diff
@@ -41,26 +42,31 @@ const HeadlessFeed = ({
     { tenant },
   );
 
-  const { items, metadata } = useNotificationStore(feedClient);
+  // Example of using a selector to access a subset of the store state.
+  const { items, metadata } = useNotificationStore(feedClient, (state) => ({
+    items: state.items,
+    metadata: state.metadata,
+  }));
```

Comment:
Does this always need a selector now?

Reply:
Nope this was just an example usage. I'll add a comment specifying that!